### PR TITLE
feat: add multiple layers example, fix typos, requiredFeatures

### PR DESCRIPTION
**Pull request by zhixiangteoh**
Fixed typos: `controller-interraction` to `controller-interaction`,
`equidirect` to `equirect`.

Added `video_in` parameter for `equirect` and `multiple-layers` apps.
Should also want to add a similar parameter `layout_in` eventually, to go
alongside supporting different video layouts corresponding to the different
videos.

Added an example for an equirect layer backdrop and a quad layer, both
displaying the same 60fps video.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,22 +1,28 @@
 import {
-    ControllerInterraction,
-    SimpleEquidirectMediaLayer,
+    ControllerInteraction,
     SimpleScene,
+    SimpleEquirectMediaLayer,
+    MultipleLayers,
 } from "./apps";
 
+import video60fps from "./media/60fps.mp4";
```
Comment (und3fined-v01d) [inline index.js:8]: This import is  not being used

**Review by und3fined-v01d — CHANGES_REQUESTED** [review]

File: apps/simple-video-layers/equirect.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ import { WebGLRenderer } from "../../util/WebGLRenderer";
 import { VRButton } from "../../util/webxr/VRButton";
 
 class App {
-    constructor() {
+    constructor(video_in = panoVideo) {
```
[the remark this answers is not in the document]
Comment (zhixiangteoh) [reply apps/simple-video-layers/equirect.js:10]: oh yes definitely; i must've been coding too much in c++ for school work...

File: index.js (hunk 1 above)
Comment (zhixiangteoh) [reply index.js:8]: Oh yes this was to test a different video. Will remove!

**Review by zhixiangteoh — COMMENTED** [review]
Rename `video_in` to `videoIn`, remove video60fps import.

**Review by und3fined-v01d — APPROVED** [review]

**Comment by und3fined-v01d** [comment]
lgtm

**Pull request merged**